### ISSUE TITLE
Jira credentials can not be saved in Jira Commit Hint plugin

### ISSUE THREAD

**Issue by benbrummer** [gitextensions/gitextensions#8397]
## Current behaviour
Settings for Jira Commit Hint plugin can be applied

## Expected behaviour
Applying the settings throws an exception

## Steps to reproduce

- Configure jira url, username and password  
- Click the apply button

## Error Details
```
System.ArgumentNullException: Value cannot be null.
Parameter name: key
   at System.Collections.Concurrent.ConcurrentDictionary`2.AddOrUpdate(TKey key, TValue addValue, Func`3 updateValueFactory)
   at GitUIPluginInterfaces.CredentialsManager.SetCredentials(SettingLevel settingLevel, String name, NetworkCredential value)
   at GitUIPluginInterfaces.CredentialsSetting.SaveValue(ISettingsSource settings, String userName, String password)
   at GitUIPluginInterfaces.CredentialsSetting.CredentialsControlBinding.SaveSetting(ISettingsSource settings, CredentialsControl control)
   at GitUIPluginInterfaces.SettingControlBinding`2.SaveSetting(ISettingsSource settings)
   at GitUI.CommandsDialogs.SettingsDialog.SettingsPageBase.PageToSettings()
   at GitUI.CommandsDialogs.SettingsDialog.SettingsPageBase.SaveSettings()
   at GitUI.CommandsDialogs.FormSettings.Save()
   at GitUI.CommandsDialogs.FormSettings.buttonApply_Click(Object sender, EventArgs e)
   at System.Windows.Forms.Control.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ButtonBase.WndProc(Message& m)
   at System.Windows.Forms.Button.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

## Additional information
Setting for Jira Hint plugin can not be saved. Preview button works. Plugin therefore can not be used at all. Saving the Jira URL field alone works. But the Jira credentials can not be applied.

## Environment
- Git Extensions 3.4.3.9999
- Build d4b0f48bbf3e39a71f5d7ff5231be5678d4aa0f1
- Git 2.28.0.windows.1
- Microsoft Windows NT 10.0.19041.0
- .NET Framework 4.8.4200.0
- DPI 96dpi (no scaling)

**Comment by gerhardol** [comment]
For me this is working OK, stored in the Windows Credential Manager. Hint can be used in Commit window.

**Comment by lukaszgradzik** [comment]
I'm having exaclty the same issue as  @benbrummer. Any solution or workaround?

**Comment by gerhardol** [comment]
Jira Commit Hint plugin is removed in 4.0, see #9659 
This issue is therefore closed as not applicable, link from #9659
